civicrm.install

```diff
@@ -32,10 +32,40 @@
  * $Id$
  *
  */
-function civicrm_install() {}
 
 /**
- * Implementation of hook_uninstall( )
+ * Enable CiviCRM.
+ */
+function civicrm_enable() {
```

Comment:
Moveing of hook_enable to .install file as per http://www.drupalcontrib.org/api/drupal/drupal%21modules%21system%21system.api.php/function/hook_enable/7 "The hook is called every time the module is enabled. It should be implemented in the module's .install file. The implementation can rely on the .module file being loaded."